src/malli/swagger.cljc

```diff
@@ -101,12 +101,12 @@
 (defmethod expand ::responses [_ v acc _]
   {:responses
    (into
-    (or (:responses acc) {})
-    (for [[status response] v]
-      [status (-> response
-                  (update :schema transform {:type :schema})
-                  (update :description (fnil identity ""))
-                  -remove-empty-keys)]))})
+     (or (:responses acc) {})
+     (for [[status response] (filter #(-> % second :schema) v)]
+       [status (-> response
+                   (update :schema transform {:type :schema})
+                   (update :description (fnil identity ""))
+                   -remove-empty-keys)]))})
```

Comment:
Please don't reindent this. We use standard clojure indentation with function arguments indented one space. Tommi just fixed this in 44a9ee07.

Reply:
Sorry my editor was misconfigured. I'll revert it.